testing/bdd/features/timing/ebuttd_resolved_timings_on_elements.feature

```diff
@@ -58,11 +61,13 @@ Feature: Resolving timings on elements
     And   it has p end time <p_end>
     When  the document is generated
     And   the EBU-TT-Live document is converted to EBU-TT-D
-    Then  p computed begin time is <p_computed_begin_time>
-    And   p computed end time is <p_computed_end_time>
+    Then  p resulted begin time is <p_resulted_begin_time>
+    And   p resulted end time is <p_resulted_end_time>
+    And   no timings present on div
+    And   no timings present on body
 
     Examples:
-      | body_begin | body_end | div_begin | div_end  | p_begin  | p_end    | p_computed_begin_time | p_computed_end_time |
+      | body_begin | body_end | div_begin | div_end  | p_begin  | p_end    | p_resulted_begin_time | p_resulted_end_time |
       | 00:01:10   |          | 00:00:01  | 00:00:04 | 00:00:02 | 00:00:03 | 00:01:13              | 00:01:14            |
       | 00:01:00   |          | 00:00:02  | 00:00:04 | 00:00:10 | 00:00:11 | 00:01:12              | 00:01:04            |
```

Comment:
I don't understand how this passes. The input `p` begins _after_ its parent `div` has ended, so it shouldn't end up in the output, and this test should fail as it stands.

It's interesting that the resulted `p` has an end time _before_ its begin time, so from a semantic perspective this is sort of correct.

Is it like this because the `p` is _not_ getting pruned even though it never ends up being active, so it needs to have some times in it just to make the thing work?